sycl/doc/design/PluginInterface.md

```diff
@@ -193,16 +193,6 @@ underlying "native" device runtimes such as OpenCL.
 There are some OpenCL interoperability APIs, which are to be implemented
 by the OpenCL PI plugin only. These APIs match semantics of the corresponding
 OpenCL APIs exactly.
```

Comment:
I think this was the only "picl" interface, so remove the entire comment/concept?

Reply:
Sentence deleted.